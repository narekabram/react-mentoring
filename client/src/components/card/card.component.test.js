import React from 'react';
import { shallow } from 'enzyme';
import Card from './card';

describe('Card', () => {
    it('Adds Card snapshot testing', () => {
        const component = shallow(<Card />);

        expect(component).toMatchSnapshot();
    });
});
