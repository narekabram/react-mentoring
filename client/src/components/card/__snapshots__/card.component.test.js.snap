// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Adds Card snapshot testing 1`] = `ShallowWrapper {}`;
